### PR TITLE
Re 232

### PR DESCRIPTION
**Pull request by cPholloway**
By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

File: lib/Elevate/OS.pm [hunk 1]
```diff
@@ -74,6 +74,7 @@ BEGIN {
         'disable_mysql_yum_repos',            # This is a list of mysql repo files to disable
         'ea_alias',                           # This is the value for the --target-os flag used when backing up an EA4 profile
         'elevate_rpm_url',                    # This is the URL used to install the leapp RPM/repo
+        'is_cloudlinux',                      # This is used to determine if the OS is CloudLinux
```
Comment (toddr) [inline lib/Elevate/OS.pm]: I'm very concerned this isn't the right question. I know cPanel has the equivalent variable. I guess I'm mostly concerned with what we're going to do with it. 

Is your real question: `provides_mysql_governor` ?

Comment (cPholloway) [reply lib/Elevate/OS.pm]: Technically, CL mysql/etc can be installed without governor being installed (one of the fun things I found out while I was investigating this).  I plan on using the variable for something similar to this:

```
    sub is_database_provided_by_cloudlinux () {
        return 0 unless Elevate::OS::is_cloudlinux();
```

That said, I could just as easily use `provides_mysql_governor` for the same purpose, but `is_cloudlinux` is more accurate imo :\

Comment (toddr) [reply lib/Elevate/OS.pm]: We have historically found that if you're testing the distro or the distro major, it's the wrong question and eventually airs as confusing in the code. ¯\_(ツ)_/¯ As an example, `provides_mysql_governor` is technically true on U20.

Comment (cPholloway) [reply lib/Elevate/OS.pm]: I'll go ahead and make this change

**Review by toddr — APPROVED** [review]

**Pull request merged**